packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -330,16 +280,43 @@ export class AtomicSearchInterface
     this.engine.executeFirstSearchAfterStandaloneSearchBoxRedirect(analytics);
   }
 
-  private engineIsCreated(engine?: SearchEngine): engine is SearchEngine {
-    if (!engine) {
-      console.error(
-        'You have to call "initialize" on the atomic-search-interface component before modifying the props or calling other public methods.',
-        this.host
+  public get bindings(): Bindings {
```

Comment:
Some not strictly needed reordering of public methods in `atomic-search-interface`.

Pet peeves of mine, but I prefer when public methods are grouped at the top of the file, and private stuff at the bottom.